### PR TITLE
Make message more helpful on failure of loading cluster default conf

### PR DESCRIPTION
**Pull request by apc999**
Fix https://github.com/Alluxio/alluxio/issues/9309.

Add "Please make sure that Alluxio master is running" to error message

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java [hunk 1]
```diff
@@ -63,7 +63,8 @@ public static void main(String[] args) {
           ServerConfiguration.getDuration(PropertyKey.WORKER_MASTER_CONNECT_RETRY_TIMEOUT)));
     } catch (IOException e) {
       ProcessUtils.fatalError(LOG,
-          "Failed to load cluster default configuration for worker: %s", e.getMessage());
+          "Failed to load cluster default configuration for worker. Please make sure that Alluxio "
+              + "master is running: %s", e.getMessage());
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java]: We should use `e.toString` so we get both the message and exception type.

Comment (apc999) [reply core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java]: fixed

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**